vm/src/stdlib/collections.rs

```diff
@@ -171,23 +177,37 @@ mod _collections {
             PyDeque {
                 deque: PyRwLock::new(self.borrow_deque().clone()),
                 maxlen: self.maxlen,
+                state: AtomicCell::new(self.state.load()),
             }
         }
 
+        fn get_cloned_vec_deque(
```

Comment:
I don't think this function is really necessary, calling `clone` on the lock should get you your `VecDeque` just fine.

Reply:
I fixed with using `clone`. Thanks!